tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -155,17 +198,20 @@ def test_streams_should_scale_in_while_brokers_down(self):
         self.wait_for_verification(processor, shutdown_message, processor.STDOUT_FILE)
         self.wait_for_verification(processor_2, shutdown_message, processor_2.STDOUT_FILE)
 
-        self.kafka.start_node(node)
+        with processor_3.node.account.monitor_log(processor_3.LOG_FILE) as monitor_3:
+            self.kafka.start_node(node)
+
+            monitor_3.wait_until(self.connected_message,
+                                 timeout_sec=120,
+                                 err_msg=("Never saw '%s' on " % self.connected_message) + str(processor_3.node.account))
 
         self.assert_produce_consume(self.inputTopic,
                                     self.outputTopic,
                                     self.client_id,
                                     "sending_message_after_stopping_streams_instance_bouncing_broker",
-                                    num_messages=9,
+                                    num_messages=self.num_messages,
                                     timeout_sec=120)
 
-        self.wait_for_verification(processor_3, "processed9messages", processor_3.STDOUT_FILE)
```

Comment:
This is just a cleanup since this line is not necessary, right?

Reply:
Yes, that's correct.